ruff_cli/src/args.rs

```diff
@@ -22,121 +22,163 @@ use rustc_hash::FxHashMap;
 pub struct Args {
     #[arg(required_unless_present_any = ["clean", "explain", "generate_shell_completion"])]
     pub files: Vec<PathBuf>,
-    /// Path to the `pyproject.toml` or `ruff.toml` file to use for
-    /// configuration.
-    #[arg(long, conflicts_with = "isolated")]
-    pub config: Option<PathBuf>,
-    /// Enable verbose logging.
-    #[arg(short, long, group = "verbosity")]
-    pub verbose: bool,
-    /// Print lint violations, but nothing else.
-    #[arg(short, long, group = "verbosity")]
-    pub quiet: bool,
-    /// Disable all logging (but still exit with status code "1" upon detecting
-    /// lint violations).
-    #[arg(short, long, group = "verbosity")]
-    pub silent: bool,
-    /// Exit with status code "0", even upon detecting lint violations.
-    #[arg(short, long)]
-    pub exit_zero: bool,
-    /// Run in watch mode by re-running whenever files change.
-    #[arg(short, long)]
-    pub watch: bool,
     /// Attempt to automatically fix lint violations.
     #[arg(long, overrides_with("no_fix"))]
     fix: bool,
+    /// Show violations with source code.
+    #[arg(long, overrides_with("no_show_source"))]
+    show_source: bool,
+    /// Avoid writing any fixed files back; instead, output a diff for each
+    /// changed file to stdout.
+    #[arg(long)]
+    pub diff: bool,
     #[clap(long, overrides_with("fix"), hide = true)]
     no_fix: bool,
+    /// Run in watch mode by re-running whenever files change.
+    #[arg(short, long)]
+    pub watch: bool,
     /// Fix any fixable lint violations, but don't report on leftover
     /// violations. Implies `--fix`.
     #[arg(long, overrides_with("no_fix_only"))]
     fix_only: bool,
     #[clap(long, overrides_with("fix_only"), hide = true)]
     no_fix_only: bool,
-    /// Avoid writing any fixed files back; instead, output a diff for each
-    /// changed file to stdout.
-    #[arg(long)]
-    pub diff: bool,
-    /// Disable cache reads.
-    #[arg(short, long)]
-    pub no_cache: bool,
-    /// Ignore all configuration files.
-    #[arg(long, conflicts_with = "config")]
-    pub isolated: bool,
+    #[clap(long, overrides_with("show_source"), hide = true)]
+    no_show_source: bool,
```

Comment:
Should this not be grouped with `show_source`? Same for `no_fix` and `fix`.

Reply:
Ah yes, good catch ... fixed :)